contracts/cw1-subkeys/src/contract.rs

```diff
@@ -147,11 +147,6 @@ pub fn check_staking_permissions(
                 return Err(ContractError::ReDelegatePerm {});
             }
         }
-        StakingMsg::Withdraw { .. } => {
```

Comment:
Hmmm... this withdraw permission just disappears?
I would add the support for the distribution messages that replace it

Reply:
FIxed.